script.js

```diff
@@ -35,47 +35,128 @@ function getHumanChoice() {
+  // Declare score values
+  // let humanScore = 0;
+  // let computerScore = 0;
+  // Loop function, run it 5 times
+  // for (let step = 0; step <5; step++) 
+  {
+  // Function that has the main game logic
   // Play game
-  const humanSelection = getHumanChoice();
+  playRound();
+  const humanSelection = humanButtonSelection;
   const computerSelection = getComputerChoice();
   // Print choices
   console.log(`Computer chose ${computerSelection} \n\Human chose ${humanSelection}`);
+  gameData.classList.add("gameData");
+  gameData.textContent = `Computer chose ${computerSelection} \n\Human chose ${humanSelection}`;
+  resultDiv.appendChild(gameData);
+  // resultDiv.textContent = `Computer chose ${computerSelection} \n\Human chose ${humanSelection}`;
   playRound(humanSelection, computerSelection);
   }
-  if (humanScore > computerScore)
-    console.log("Human won the game, congratulations!");
-  else if (computerScore > humanScore)
-    console.log("Computer won the game, better luck next time human!");
-  else console.log("It's a tie. Give it another go to get a winner!");
+  if (humanScore >= 5 || computerScore >= 5) {
+    if (humanScore > computerScore) {
+      console.log("Human won the game, congratulations!");
+      gameData.classList.add("gameData");
+      gameData.textContent = "Human won the game, congratulations!";
+      resultDiv.appendChild(gameData);
+      // resultDiv.textContent = "Human won the game, congratulations!";
+    }
+    else if (computerScore > humanScore) {
+      console.log("Computer won the game, better luck next time human!");
+      gameData.classList.add("gameData");
+      gameData.textContent =
+        "Computer won the game, better luck next time human!";
+      resultDiv.appendChild(gameData);
+      // resultDiv.textContent = "Computer won the game, better luck next time human!";
+    }
+    else {
+      console.log("Computer won the game, better luck next time human!");
+      gameData.classList.add("gameData");
+      gameData.textContent = "It's a tie. Give it another go to get a winner!";
+      resultDiv.appendChild(gameData);
+      // resultDiv.textContent = "It's a tie. Give it another go to get a winner!";
+    }
+  }
+}
+// playGame(humanButtonChoice);
+// console.log("Game end");
+const rockbtn = document.querySelector("#btn-rock");
+function onRockClick(){
+  // console.log("Rock btn");
+  playGame("rock");
+}
+// rockbtn.addEventListener("click", () => {
+//   console.log("Rock btn");
+  // playRound("rock");
+// })
+const paperbtn = document.querySelector("#btn-paper");
+function onPaperClick(){
+  // console.log("Paper btn");
+  playGame("paper");
+}
+const scissorsbtn = document.querySelector("#btn-scissors");
+function onScissorsClick(){
+  // console.log("Scissors btn");
+  playGame("scissor");
 }
-playGame();
-// console.log("Game end");
\ No newline at end of file
+const resultDiv = document.querySelector("#result");
+const gameData = document.createElement("div");
```

Comment:
Move those two above all the functions that uses them.

It's good practice to define all global variables as early as possible, so it's clear later in the code what those do and where they come from.